src/rulesets/Base/fastmath_able.jl

```diff
@@ -236,14 +236,48 @@ let
         end
```

Comment:
I didn't write an frule. But perhaps this should be just `∂xy = muladd(Δx, y, x .* Δy)` without the dot, now?

Reply:
sounds right to me

Reply:
Done.